pyo3-macros-backend/src/pyclass.rs

```diff
@@ -1091,8 +1091,8 @@ fn impl_complex_enum_struct_variant_cls(
         let field_getter_impl = quote! {
             fn #field_name(slf: #pyo3_path::PyRef<Self>) -> #pyo3_path::PyResult<#field_type> {
                 match &*slf.into_super() {
-                    #enum_name::#variant_ident { #field_name, .. } => Ok(#field_name.clone()),
-                    _ => unreachable!("Wrong complex enum variant found in variant wrapper PyClass"),
+                    #enum_name::#variant_ident { #field_name, .. } => ::std::result::Result::Ok(#field_name.clone()),
```

Comment:
```suggestion
                    #enum_name::#variant_ident { #field_name, .. } => ::std::result::Result::Ok(::std::clone::Clone::clone(&#field_name)),
```

Reply:
Ahhh,  I see now!  I wasn't quite following the error messages before, now they make sense. :smile: .  I have pushed the fixes for from and clone.  Thanks!